src/containers/App.jsx

```diff
@@ -90,47 +91,55 @@ class App extends Component {
     const isFetchingContext = status === FETCHING_CONTEXT
 
     const isReady = !hasError && !isFetching && !isFetchingContext
+    const showTimeline = flag('hide_konnector_errors')
```

Comment:
The next demo is soon, I'll do another PR tonight to harmonize the feature flag names.

Reply:
just add a todo then :p